lunamark/reader/markdown.lua

```diff
@@ -1386,7 +1583,7 @@ function M.new(writer, options)
         inp = rest
       end
       local result = { writer.start_document(), parse_blocks(inp), writer.stop_document() }
-      return rope_to_string(result), writer.get_metadata()
+      return writer.merge(result), writer.get_metadata()
```

Comment:
Please explain.

Reply:
If this is accepted, maybe the `merge()` name is ill-chosen, though, and not that self-explaining.  Perhaps simply `output`, or even `ropes_to_output` ?

Reply:
For now, I opted for `rope_to_ouput` (singular), since the default just calls `util.rope_to_string`)